packages/mobile/src/components/details-tile/DetailsTileActionButtons.tsx

```diff
@@ -89,6 +104,27 @@ export const DetailsTileActionButtons = ({
   const { isEnabled: isPlaylistUpdatesEnabled } = useFeatureFlag(
     FeatureFlags.PLAYLIST_UPDATES_PRE_QA
   )
+  const collection = useSelector((state: CommonState) =>
```

Comment:
can we merge all these selectors into a single big one? ideally also the "isCollectionEmpty" and "collectionHasPrivateTrack" booleans can be tied up in those selectors so the rerenders are really minimized? lmk if you want to pair on some ideas here. maybe a "getCollectionTracks" selector, and then we have two selectors that call that + do the final boolean check?